Frontend/CO.CDP.OrganisationApp.Tests/AuthorizationTests.cs

```diff
@@ -0,0 +1,212 @@
+                }
+            );
+
+        organisationClient.Setup(client => client.GetOrganisationAsync(testOrganisationId))
+            .ReturnsAsync(
+                new Organisation.WebApiClient.Organisation(
+                    new List<Identifier> { },
+                    new List<Address> { },
+                    new ContactPoint("a@b.com", "Contact", "123", new Uri("http://whatever")),
+                    testOrganisationId,
+                    new Identifier("asd", "asd", "asd", new Uri("http://whatever")),
+                    "Org name",
+                    new List<Organisation.WebApiClient.PartyRole> { Organisation.WebApiClient.PartyRole.Supplier, Organisation.WebApiClient.PartyRole.ProcuringEntity }
+                )
+            );
+
+        services.AddTransient<IOrganisationClient, OrganisationClient>(sc => organisationClient.Object);
+
+        _mockSession.Setup(s => s.Get<Models.UserDetails>(Session.UserDetailsKey))
+            .Returns(new Models.UserDetails() { Email = "a@b.com", UserUrn = "urn", PersonId = person.Id });
+
+        services.AddSingleton(_mockSession.Object);
+
+        services.AddSingleton<IPolicyEvaluator, FakeAuthenticationPolicyEvaluator>();
+
+        services.AddAuthentication(options =>
+        {
+            options.DefaultScheme = CookieAuthenticationDefaults.AuthenticationScheme;
+            options.DefaultChallengeScheme = OpenIdConnectDefaults.AuthenticationScheme;
+        })
+        .AddScheme<AuthenticationSchemeOptions, FakeCookieAuthHandler>(CookieAuthenticationDefaults.AuthenticationScheme, options => { });
+
+        services.Configure<OpenIdConnectOptions>(OpenIdConnectDefaults.AuthenticationScheme, options => {
+            options.ClientId = "123";
+            options.Authority = "https://whatever";
+        });
+
+        var factory = new CustomisableWebApplicationFactory<Program>(services);
+
+        return factory.CreateClient();
+    }
+
+    public static IEnumerable<object[]> TestCases()
+    {
+        yield return new object[] { $"/organisation/{testOrganisationId}/users/user-summary", new string[] { "Organisation has 2 users" } };
+        yield return new object[] { $"/organisation/{testOrganisationId}/users/{personInviteGuid}/change-role?handler=personInvite", new string[] { "Person invite Last name", "Can add, remove and edit users" } };
+    }
+
+    [Theory]
+    [MemberData(nameof(TestCases))]
+    public async Task TestAuthorizationIsSuccessful_WhenUserIsAllowedToAccessResourceAsAdminUser(string url, string[] expectedTexts)
+    {
+        var _httpClient = BuildHttpClient(new List<string> { OrganisationPersonScopes.Admin });
+
+        var request = new HttpRequestMessage(HttpMethod.Get, url);
+
+        var response = await _httpClient.SendAsync(request);
+
+        var responseBody = await response.Content.ReadAsStringAsync();
+
+        Assert.NotNull(responseBody);
```

Comment:
As a team we made decision to use [FluentAssertions ](https://fluentassertions.com/introduction)in unit tests.
Recently this was reminded on slack channel as well as in another PR review.
You can get reference from another unit tests from the repo, or feel free to speak any member of dev team for further query.

Reply:
Done 👍🏻